theme/templates/resource-landing-page/title-section.html

```diff
@@ -371,35 +384,101 @@ <h2 id="resource-title">{{ title }}</h2>
         }
     },
     {% endif %}
-    "creator": [
```

Comment:
This block is more clear to me how the json will look like than building it in hydroshare_tags.

Reply:
The original logic was not that complicated for creating creators element, which was why I did it in the template directly initially. However, when addressing Jeff's feedback, I realized I needed to add more and more logic into creating creators json-ld element, and at some point along the way, the logic was not manageable to do it directly in the template any more, which was when I started to do the logic in the template tag.